network.go

```diff
@@ -10,18 +10,23 @@ import (
 
 // Network represents a private network for instances to connect to
 type Network struct {
-	ID            string            `json:"id"`
-	Name          string            `json:"name,omitempty"`
-	Default       bool              `json:"default"`
-	CIDR          string            `json:"cidr,omitempty"`
-	CIDRV6        string            `json:"cidr_v6,omitempty"`
-	Label         string            `json:"label,omitempty"`
-	Status        string            `json:"status,omitempty"`
-	IPv4Enabled   bool              `json:"ipv4_enabled,omitempty"`
-	IPv6Enabled   bool              `json:"ipv6_enabled,omitempty"`
-	NameserversV4 []string          `json:"nameservers_v4,omitempty"`
-	NameserversV6 []string          `json:"nameservers_v6,omitempty"`
-	VLAN          VLANConnectConfig `json:"vlan,omitempty"`
+	ID                    string   `json:"id"`
+	Name                  string   `json:"name,omitempty"`
+	Default               bool     `json:"default"`
+	CIDR                  string   `json:"cidr,omitempty"`
+	CIDRV6                string   `json:"cidr_v6,omitempty"`
+	Label                 string   `json:"label,omitempty"`
+	Status                string   `json:"status,omitempty"`
+	IPv4Enabled           bool     `json:"ipv4_enabled,omitempty"`
+	IPv6Enabled           bool     `json:"ipv6_enabled,omitempty"`
+	NameserversV4         []string `json:"nameservers_v4,omitempty"`
+	NameserversV6         []string `json:"nameservers_v6,omitempty"`
+	VlanID                int      `json:"vlan_id" validate:"required" schema:"vlan_id"`
+	HardwareAddr          string   `json:"hardware_addr,omitempty" schema:"hardware_addr"`
+	CIDRv4                string   `json:"cidr_v4" validate:"required" schema:"cidr_v4"`
```

Comment:
We have cidr and cidr_v4?

Reply:
https://github.com/civo/civogo/pull/191/commits/fe7b76fe62370d2baf56a2dab7440e468d92c27f